src/metal.cs

```diff
@@ -1741,6 +2001,69 @@ partial interface MTLRenderCommandEncoder : MTLCommandEncoder {
 #endif
 		[Export ("drawIndexedPatches:patchIndexBuffer:patchIndexBufferOffset:controlPointIndexBuffer:controlPointIndexBufferOffset:indirectBuffer:indirectBufferOffset:")]
 		void DrawIndexedPatches (nuint numberOfPatchControlPoints, [NullAllowed] IMTLBuffer patchIndexBuffer, nuint patchIndexBufferOffset, IMTLBuffer controlPointIndexBuffer, nuint controlPointIndexBufferOffset, IMTLBuffer indirectBuffer, nuint indirectBufferOffset);
+
+		[Mac (10,13), NoiOS, NoTV, NoWatch]
+#if XAMCORE_4_0
+		[Abstract]
+#endif
+		[Export ("setViewports:count:")]
+		void SetViewports (IntPtr viewports, nuint count);
+
+		[Mac (10,13), NoiOS, NoTV, NoWatch]
+#if XAMCORE_4_0
+		[Abstract]
+#endif
+		[Export ("setScissorRects:count:")]
+		void SetScissorRects (IntPtr scissorRects, nuint count);
```

Comment:
As above, MTLScissorRect is a struct, not an NSObject, so with that binding the generator wants to convert it to an NSArray of NSObjects, which fails (and doesn't match the selector anyway):

```
- (void)setScissorRects:(const MTLScissorRect *)scissorRects 
                  count:(NSUInteger)count;
```